src/Core/View.js

```diff
@@ -495,6 +511,10 @@ View.prototype.execFrameRequesters = function execFrameRequesters(when, dt, upda
             frameRequester(dt, updateLoopRestarted, args);
         }
     }
+
+    if (this._delayedFrameRequesterRemoval.length > 0) {
```

Comment:
I'm pretty sure this should be at the beginning of `execFrameRequesters`, otherwise if you remove a frame requester outside of this `step` context, it will execute once more before being removed.

Reply:
Done